core/src/main/java/org/wildfly/extras/creaper/core/online/SslOptions.java

```diff
@@ -174,6 +177,14 @@ public Builder trustStoreType(KeyStoreType trustStoreType) {
             return this;
         }
 
+        /**
+         * Errors during hostname verification will be ignored.
+         */
+        public Builder turnOffHostnameVerification() {
```

Comment:
Nothing more than you have written -- there is no such thing as hostname verification for the native management protocols.